src/encoder.rs

```diff
@@ -98,66 +120,47 @@ impl Component for Encoder {
-                                    <option value="0" selected=true>{ "Polkadot" }</option>
-                                    <option value="2">{ "Kusama" }</option>
-                                    <option value="42">{ "Substrate" }</option>
-                                    <option value="137">{ "Vara" }</option>
-                                    <option value="custom">{ "Custom" }</option>
-                                </select>
-                            </div>
-                        </div>
-                        <div class="control is-expanded">
-                            <input class="input is-info" type="number" min="0" max="16383"
-                                disabled={ !self.custom_prefix } ref={ self.prefix_ref.clone() } value={ self.prefix.to_string() }/>
+        <div hidden={ !ctx.props().active }>
+            <div class="field">
+                <label class="label">{ "Public Key" }</label>
+                { self.key_field() }
+                { self.error_help() }
+            </div>
+            <div class="field has-addons">
+                <span class="control p-1">
+                    <div class="pretty p-switch p-fill">
+                        <input checked={ self.custom_prefix } type="checkbox" ref={ self.checkbox_ref.clone() }
+                            onclick={ ctx.link().callback(|_| Msg::Prefix) } />
+                        <div class="state p-primary">
+                            <label>{ "Custom Prefix"} </label>
+                            // <label>Primary</label>
                         </div>
                     </div>
-                </div>
 
-                <div class="field">
-                    <label class="label">{ "Public Key" }</label>
-                    { self.key_field() }
-                    { self.error_help() }
-                </div>
-
-                <div class="buttons">
-                    <button class="button is-info is-primary"
-                        onclick={ ctx.link().callback(|_| Msg::Convert) }>
-                        <span class="icon"><i class="fas fa-sync"></i></span>
-                        <span>{ "Convert" }</span>
-                    </button>
-                    <button class="button is-light" onclick={ ctx.link().callback(|_| Msg::Alice) }>
-                        <span class="icon"><i class="fas fa-user"></i></span>
-                        <span>{ "Alice" }</span>
-                    </button>
-                    <button class="button is-danger" onclick={ ctx.link().callback(|_| Msg::Clear) }>
-                        <span class="icon"><i class="fas fa-times"></i></span>
-                        <span>{ "Clear" }</span>
-                    </button>
+                </span>
+                <div class="control p-1">
+                    <input class="input is-info is-small" type="number" min="0" max="16383" disabled={ !self.custom_prefix }
```

Comment:
As for me, regular input looks better
```suggestion
                    <input class="input is-info" type="number" min="0" max="16383" disabled={ !self.custom_prefix }
```

Reply:
<img width="486" alt="image" src="https://github.com/gear-tech/ss58.org/assets/78953304/36a9875a-b3d3-4abe-8da6-c4c24aafe070">

I think the regular size is a little distracting